Update quickstart-create-portal.md (#124829)
Azure/azure-content: articles/azure-compute-fleet/quickstart-create-portal.md

# Create an Azure Compute Fleet using Azure portal (Preview)

> [!IMPORTANT]

> > Azure Compute Fleet is currently in preview. Previews are made available to you if you agree to the [supplemental terms of use](https://azure.microsoft.com/support/legal/preview-supplemental-terms/). Some aspects of this feature may change before general availability (GA). 

Reviewer: ```suggestion
> Azure Compute Fleet is currently in preview. Previews are made available to you on the condition that you agree to the [supplemental terms of use](https://azure.microsoft.com/support/legal/preview-supplemental-terms/). Some aspects of this feature may change prior to general availability (GA). 
```

Reviewer: Please revert back. This disclaimer language is used across Compute docs and we need it to match across all the docs. Thank you. I'll take the feedback you gave to the team and we may update the wording in the future, but for now it must stay as is.

Reviewer 2: With this feedback, I will close the PR. If this article requires further changes, please open a separate pull request. #please-close
